csharp/test/Ice/scope/Server.cs

```diff
@@ -23,7 +23,8 @@ public override Test.MyStruct[]
         }
 
         public override Dictionary<string, Test.MyStruct>
-        opMyStructMap(Dictionary<string, Test.MyStruct> s1, out Dictionary<string, Test.MyStruct> s2, Ice.Current current)
+        opMyStructMap(Dictionary<string, Test.MyStruct> s1,
+            out Dictionary<string, Test.MyStruct> s2, Ice.Current current)
```

Comment:
```suggestion
        opMyStructMap(
            Dictionary<string, Test.MyStruct> s1,
            out Dictionary<string, Test.MyStruct> s2, Ice.Current current)
```

Reply:
each parameter should be on its own line in this case